app/load_data/load_ndcs.py

```diff
@@ -89,7 +94,9 @@ def clear_tables(session):
 
 
 def main(session):
-    f = open('data/drug-ndc-20200504.json')
+    p = Path.cwd()
+    p = (p / 'data')
+    f = open(p / 'drug-ndc.json')
```

Comment:
This introduces a subtle bug with `.gitignore`. Should likely ignore the whole data directory. 

We should also consider using a config file instead of `Path.cwd`, something mentioned by @hl1. Not necessary to do immediately but i think it would be good down the road